plugins/plugin-chart-echarts/src/BoxPlot/buildQuery.ts

```diff
@@ -22,14 +22,14 @@ import { BoxPlotQueryFormData, BoxPlotQueryObjectWhiskerType } from './types';
 const PERCENTILE_REGEX = /(\d+)\/(\d+) percentiles/;
 
 export default function buildQuery(formData: BoxPlotQueryFormData) {
-  const { whiskerOptions, columns: distributionColumns = [] } = formData;
+  const { columns = [], granularity_sqla, groupby = [], whiskerOptions } = formData;
   return buildQueryContext(formData, baseQueryObject => {
     let whiskerType: BoxPlotQueryObjectWhiskerType;
     let percentiles: [number, number] | undefined;
-    const { columns = [], metrics = [] } = baseQueryObject;
+    const { metrics = [] } = baseQueryObject;
     const percentileMatch = PERCENTILE_REGEX.exec(whiskerOptions as string);
 
-    if (whiskerOptions === 'Tukey') {
+    if (whiskerOptions === 'Tukey' || whiskerOptions === undefined) {
```

Comment:
![image](https://user-images.githubusercontent.com/15073128/132855555-4ef20ad3-234a-47cb-a950-76e965abe50f.png)

Should we check for null instead of undefined?

Reply:
Oh, good catch! Although we should probably make the whisker type mandatory. Anyhoo, needs to be fixed 👍

Reply:
I made it unclearable and changed this to check for `!whiskerOptions` to ensure backwards compatibility in case someone has saved it with an empty whisker type.